pkg/services/tree/service.proto

```diff
@@ -47,6 +48,8 @@ service TreeService {
   rpc Apply (ApplyRequest) returns (ApplyResponse);
   // GetOpLog returns a stream of logged operations starting from some height.
   rpc GetOpLog(GetOpLogRequest) returns (stream GetOpLogResponse);
+  // Healthcheck is a dummy rpc to check is service available
```

Comment:
please, double check that the comment is correct, seems like it should be `check if ... is ...` construct

Reply:
Thanks, rephrased.